src/script/util/valueStore.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+/**
+ * The value store util allows storing a single value across page navigation.
+ * This value will never be stored on the disk, which makes it a good candidate for storing cryptographic key.
+ * There are no guarantees regarding the time the value will stay in memory. Should not be used as a reliable store.
+ * Main use case is to pass on a value from a page to another page.
+ */
+export interface ValueStoreApi {
+  get: () => Promise<any>;
+  save: (value: any) => Promise<any>;
```

Comment:
Is it possible to type this?

Reply:
You can literally store anything you want. I think any is the proper type here. Any suggestions ?

Reply:
The question is, e.g. does it return the `value`? Then we could replace `any` with a generic.

Reply:
Oh, right !